rust/lance-encoding/src/encodings/physical/rle.rs

```diff
@@ -1016,4 +955,79 @@ mod tests {
             }
         }
     }
+
+    // ========== Encoding Verification Tests ==========
+
+    #[test_log::test(tokio::test)]
+    async fn test_rle_encoding_verification() {
```

Comment:
Do we need a new test or should we just add `expected_encoding` to the existing tests?